orm/encoding/ormfield/string.go

```diff
@@ -16,15 +16,15 @@ func (s StringCodec) FixedBufferSize() int {
 }
 
 func (s StringCodec) ComputeBufferSize(value protoreflect.Value) (int, error) {
-	return len(value.Interface().(string)), nil
+	return len(value.String()), nil
```

Comment:
Interface().(string) is more correct unless there's 100% guarantee that this will always be a string (no invalid values either)

Reply:
The framework will never call this with a non string value. String is more performant

We can make this package internal if you think there's a risk

Reply:
I think we can safely assume that all any type used in ORM correctly implements the .String() method